server/webapp/WEB-INF/rails.new/app/assets/javascripts/pipeline_configs/models/materials.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-define(['mithril', 'lodash', 'string-plus', './model_mixins', './encrypted_value'], function (m, _, s, Mixins, EncryptedValue) {
+define(['mithril', 'lodash', 'string-plus', 'jquery',  './model_mixins', './encrypted_value', 'js-routes'], function (m, _, s, $, Mixins, EncryptedValue, Routes) {
 
```

Comment:
I think we should use m.request
Here's pointers on how to make it testable
http://stackoverflow.com/questions/26373734/how-can-i-unit-test-a-model-using-m-request-in-mithril